src/index.html

```diff
@@ -12,7 +13,49 @@
       href="./style.css"
     />
```

Comment:
HTML Formatting: The attributes of your `link` tag are not correctly formatted. Each attribute should start on a new line with a 2-space indentation relative to the tag.